derive_builder/src/options/mod.rs

```diff
@@ -198,6 +206,9 @@ impl<Mode> OptionsBuilder<Mode> where
                 // setter implicitly enabled
                 self.setter_enabled(true)
             },
+            "try_setter" => {
+                self.try_setter_enabled(true)
```

Comment:
please set `setter_enabled` implicitly if the user specifies `#[builder(try_setter)]`
```rust
"try_setter" => {
    /* ... */
    // setter implicitly enabled
    self.setter_enabled(true)
}
```